test/__init__.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
Move all this code to test/py/

Reply:
As per our last discussion I have moved all code to /test/py/ file except pytest.ini file, as this pytest.ini file must be at /cortx-halo/ directory otherwise pytest will not able to find configs ( i.e. pytest.ini ) & start throwing warnings "PytestUnknownMarkWarning".